client/rest-exchangers.go

```diff
@@ -0,0 +1,121 @@
+/*
+Currency implementation is abstractly support multiple exchangers functionality, so API user can compare
+available exchanger services and choose any.
+For not we support only Changelly exchanger 3rd party.
+ */
```

Comment:
at the top of any code file we need a short header about license

Reply:
done